seatunnel-core/seatunnel-core-spark/src/main/java/org/apache/seatunnel/core/spark/SparkStarter.java

```diff
@@ -191,11 +191,15 @@ private List<Path> getPluginsJarDependencies() throws IOException {
         if (!Files.exists(pluginRootDir) || !Files.isDirectory(pluginRootDir)) {
             return Collections.emptyList();
         }
+        //my test code
+        Path pluginJarPath = Paths.get("/Users/chenhu/projects/org/apache/seatunnel/seatunnel-common/plugins/oracle/lib/ojdbc8.jar");
+
```

Comment:
This path seems to be problematic, not general enough, please modify it, thanks

Reply:
o, my mistake